plugins/SitesManager/tests/Unit/APITest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\SitesManager\tests\Unit;
+
+use Piwik\Container\StaticContainer;
+use Piwik\Plugins\SitesManager\API;
+use Piwik\SettingsServer;
+use Piwik\Translate;
+
+/**
+ * @group SitesManaager
+ * @group APITest
+ * @group Plugins
+ */
+class APITest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var Api
+     */
+    private $api;
+
+    public function setUp()
+    {
+        parent::setUp();
+
+        if (!SettingsServer::isTimezoneSupportEnabled()) {
+            $this->markTestSkipped('timezones needs to be supported');
+        }
+
+        Translate::loadAllTranslations();
```

Comment:
Would it be possible to not use translations in this test and instead assert that the translation keys are what we expect?

Reply:
Not really. This would make e.g. “UTC+1” be returned as “Intl_GmtFormat” (the offset placeholder would be ignores), and we wouldn't be able to test the distinction between known and unknown cities (Asia/Foo_Bar vs. America/New_York).